config/default.json

```diff
@@ -41,5 +41,9 @@
 	},
 	"CTL_TOOLS": {
 		"EXTERNAL_TOOL_MAX_LOGO_SIZE_IN_BYTES": 300000
+	},
+	"ADMIN_API": {
+		"ENABLED": true,
+		"ALLOWED_API_KEYS": "1ab2c3d4e5f61ab2c3d4e5f6"
```

Comment:
Please set it to an empty string by default - this way it should not block the application from starting (if it would be not set in the running env) and also will not give any access to the Admin API to anyone by an accident.